packages/core/src/controllers/relayer.ts

```diff
@@ -19,7 +19,7 @@ import {
   Logger,
 } from "@walletconnect/logger";
 import { RelayJsonRpc } from "@walletconnect/relay-api";
-import { FIVE_MINUTES, ONE_SECOND, THIRTY_SECONDS, toMiliseconds } from "@walletconnect/time";
+import { FIVE_MINUTES, FIVE_SECONDS, THIRTY_SECONDS, toMiliseconds } from "@walletconnect/time";
```

Comment:
```suggestion
import { FIVE_MINUTES, ONE_SECOND, FIVE_SECONDS, THIRTY_SECONDS, toMiliseconds } from "@walletconnect/time";
```
could you re-add the `ONE_SECOND` constant as its used in other places around the relayer